shell/src/main/java/alluxio/shell/command/FreeCommand.java

```diff
@@ -58,11 +58,11 @@ void runCommand(AlluxioURI path, CommandLine cl) throws IOException {
 
   @Override
   public String getUsage() {
-    return "free <file path|folder path>";
+    return "free <path>";
   }
 
   @Override
   public String getDescription() {
-    return "Removes the file or directory(recursively) from Alluxio memory space.";
+    return "Frees the space of a file or directory(recursively) from Alluxio memory space.";
```

Comment:
Actually I would reword this to: `Frees the space occupied by a file or a directory in Alluxio.`